CHANGELOG.md

```
@@ -0,0 +1,4 @@
+# Change Log
+
+## 2025-05-25
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)